### PR TITLE
Fix z-index & wrong CSS

### LINKED ISSUE #1095: (dialog): Wrong z-index
### What MDC-Web Version are you using?

> 0.17.0

### What browser(s) is this bug affecting?

> All browsers

### What OS are you using?

> All OS (Android 7, Windows 7 confirmed)

### What are the steps to reproduce the bug?

> 1. Go to http://material-components-web.appspot.com/dialog.html
> 2. Open dialog
> 3. Observe the component's behavior

### What is the expected behavior?

> There should be a dark overlay over the toolbar

### What is the actual behavior?

> Dialogs z-index is 2 and there's no dark overlay over the toolbar

### PR DESCRIPTION
**Pull request by touficbatache**
Fixes and closes #1095 #1096

### CODE REVIEW

File: packages/mdc-dialog/README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ They only need to match the values set for their corresponding aria attributes.
 
 Dialog actions use system colors by default, but you can use a contrasting color, such as the palette’s accent color, to distinguish dialog actions from dialog content. To emphasize an action from other contents, add `mdc-dialog__action` to `mdc-button` to apply accent color.
 
-```
+```html
 <aside class="mdc-dialog">
   <div class="mdc-dialog__surface">
```
Comment (trimox) [inline packages/mdc-dialog/README.md]: Needs closing `</div>` in example

Comment (touficbatache) [inline packages/mdc-dialog/README.md]: Thanks. Just fixed it.

**Comment by touficbatache** [comment]
No problem, happy to be a part of the community!

**Comment by touficbatache** [comment]
@lynnjepsen Should this be merged? :smiley:?

**Comment by lynnjepsen** [comment]
I'd still like an example of a dialog with an action button added to our demo page. Then we can merge it.

**Comment by touficbatache** [comment]
@lynnjepsen Sorry, I thought I added the demo but it turns out I didn't. Just added it :smiley:

**Comment by touficbatache** [comment]
No problem :)

**Pull request merged**